### DISCUSSION TITLE
How can I determine minimum visible altitude for each position of a raster?

### DISCUSSION THREAD

**Discussion by sebhan2** [makepath/xarray-spatial#385] [Q&A]
Hello,
I have the following problem.

I have an observer position on a raster with a fixed height above ground. I want to know for each position on the raster the minimum altitude at which the observer is able to see an object.

How can I determine minimum visible altitude for each position of a raster?

Thank you.

Regards

**Comment by brendancol** [comment]
@sebhan2 sorry for the delayed response. We have an experimental tool here and would love feedback on it: https://github.com/xarray-contrib/xarray-spatial/blob/master/xrspatial/experimental/min_observable_height.py
